src/APIFunctions/OfficerManager.js

```diff
@@ -0,0 +1,87 @@
+import axios from 'axios';
+import { ApiResponse } from './ApiResponses';
+
+export async function createOfficer(newOfficer, token){
+  let status = new ApiResponse();
+  const officerToAdd = {
+    name: newOfficer.name,
+    email: newOfficer.email,
+    linkedin: newOfficer.linkedin,
+    team: newOfficer.team,
+    position: newOfficer.position,
+    quote: newOfficer.quote,
+    pictureUrl: newOfficer.pictureUrl
+  };
+  await axios
+    .post('api/officerManager/submit', {token, ...officerToAdd})
```

Comment:
do you need to declare a new `officerToAdd` variable above? it seems like its identical to `newOfficer`. if that's the case you can do:
```suggestion
    .post('api/officerManager/submit', {token, ...newOfficer})
```